doc/news/changes.h

```diff
@@ -87,6 +87,12 @@ inconvenience this causes.
 <a name="specific"></a>
 <h3>Specific improvements</h3>
 
+<ol> 
+ <li> New: SphericalManifold() now implements the new interface of ChartManifold()
+ for tangent vectors..
```

Comment:
double period.

In fact, I had meant not to write an entry there yet but instead, once we're finished with all of these tangent implementations, just write something about all of it in the "general" section.